lib/registration.pm

```diff
@@ -731,8 +731,11 @@ sub yast_scc_registration {
     # for leap to sle migration, we need to install yast2-registration
     # before running yast2 registration module.
     my $client_module = 'scc';
+    # For leap to SLES migratoin, we need install yast2-registration and enable
```

Comment:
install yast2-registration => install yast2-registration and rollback-helper

Reply:
Changed.